main.go

```diff
@@ -238,6 +240,16 @@ func main() { //nolint:funlen,maintidx
 		// if you are doing or is intended to do any operation such as perform cleanups
 		// after the manager stops then its usage might be unsafe.
 		// LeaderElectionReleaseOnCancel: true,
+		Client: client.Options{
+			Cache: &client.CacheOptions{
+				DisableFor: []client.Object{
+					resources.GvkToUnstructured(gvk.OpenshiftIngress),
```

Comment:
Probably something to investigated, this has been added because otherwise it would trigger the controller-runtime to start a `list-watch` but for this resource it is not needed.